docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+phpdoc:
+  container_name: phpdoc
+  image: phpdoc:latest
+  environment:
+    SYMFONY_DEBUG: "true"
+    SYMFONY_ENV: "dev"
+    SYMFONY__KERNEL__CACHE_DIR: "/tmp/cache"
+    SYMFONY__KERNEL__LOGS_DIR: "/tmp/logs"
+  volumes:
+      - /opt/webapp
+      - ./docker/vhost.conf:/etc/apache2/sites-available/000-default.conf
+      - ./docker/development:/etc/apache2/sites-available/development.override
+  working_dir: "/opt/webapp"
+  ports: [ "8000:80" ]
+  links:
+    - mysql
+
+mysql:
+    image: mysql
+    ports:
+      - "3306:3306"
+    environment:
+      - MYSQL_DATABASE=phpdoc
```

Comment:
This way we could also move the parameters.yml stuff into environment variables and make sure those are in sync